---
title: "**Microbiota predict _Clostridium difficile_ severity in germ-free mice colonized with human feces**"
bibliography: references.bib
output:
  pdf_document:
    includes:
      in_header: header.tex
    keep_tex: yes
csl: plos.csl #Get themes at https://github.com/citation-style-language/styles
fontsize: 11pt
geometry: margin=1.0in
---

\vspace{35mm}

Running title: Microbiota predict _C. difficile_ severity in humanized mice

\vspace{35mm}


Kaitlin J. Flynn^1+^, Nicholas Lesniak^1+^, Alyxandria M. Schubert^2^, Hamide Sinani^?^, and Patrick D. Schloss^1$\dagger$^

\vspace{35mm}

$\dagger$ Corresponding author: pschloss@umich.edu

\+ These authors contributed equally to this work

1\. Department of Microbiology and Immunology, University of Michigan, Ann Arbor, Michigan 48109

2\. Food and Drug Administration?

3\. Department for Hamide?


\newpage
\linenumbers


### Abstract
<<<<<<< HEAD
_Clostridium difficile_ causes diarrheal disease when it successfully colonizes a dysbiotic gut microbial community. Current mouse models to study _C. difficile_ infection (CDI) rely on pre-treatment with antibiotics to disrupt the mouse microbiome prior to inoculation. This model does not allow for analysis of human-associated microbial community members that modulate _C. difficile_ colonization and expansion. To study human-associated microbes in the context of CDI, we inoculated germ-free C57BL/6 mice with one of 16 human fecal samples from diarrheal or healthy donors and challenged with _C. difficile_ 14 days later. Five unique donor-mice combinations resulted in severe CDI while the remaining 11 only experienced mild disease. Both healthy and diarrheal donors were susceptible to colonization and severe symptoms of CDI. To determine if specific microbes were associated with disease severity outcomes, we built a classification Random Forest machine learning model based on relative abundance data of the communities prior to infection. The model identified a number of bacterial populations associated with the development of severe CDI, including _Bacilliales, Ruminococcaceae, Ruminococcus, Staphylococcus, Streptococcus and Bacteriodetes_. Additionally, a regression model accurately predicted colonization levels of _C. difficile_ at one to ten days post-infection. This model explained 86% of the variance in the number of CFU isolated from mouse stool. Members of _Lachnospiraceae, Parabacteroides, Bacteroidales, Bacteroidetes, Porphyromonadaceae_ and unclassified _Bacteria_ families were predictive of future _C. difficile_ colonization levels. Finally, challenging these mice with different strains of _C. difficile_ revealed that susceptible human-associated microbial communities were prone to severe disease independent of strain type. Taken together these results suggest that human-associated microbial communities can be recapitulated in germ-free mice and used to characterize dynamics of CDI. Because both healthy and diarrheal patients were susceptible to severe CDI, machine-learning models are useful to identify bacterial populations that allow colonization and contribute to the development of _C. difficile_ associated disease in humans. 
=======
_Clostridium difficile_ causes diarrheal disease when it successfully colonizes a disrupted gut microbial community. Current mouse models to study _C. difficile_ infection (CDI) rely on pre-treatment with antibiotics to alter the mouse microbiome prior to inoculation. This model does not allow for analysis of human-associated microbial community members that modulate _C. difficile_ colonization and expansion. To study human-associated microbes in the context of CDI, we inoculated germ-free C57BL/6 mice with one of 16 human fecal samples from diarrheal or healthy donors and challenged with _C. difficile_ 14 days later. Five unique donor-mice combinations resulted in severe CDI while the remaining 11 only experienced mild disease. Both healthy and diarrheal donors were susceptible to colonization and severe symptoms of CDI. To determine if specific microbes were associated with disease severity outcomes, we built a classification Random Forest machine learning model based on relative abundance data of the communities prior to infection. The model identified a number of bacterial populations associated with the development of severe CDI, including _Bacilliales, Ruminococcaceae, Ruminococcus, Staphylococcus, Streptococcus and Bacteriodetes_. Additionally, a regression model accurately predicted colonization levels of _C. difficile_ at one to ten days post-infection. This model explained 99% of the variance in the number of CFU isolated from mouse stool. Members of _Lachnospiraceae, Parabacteroides, Bacteroidales, Bacteroidetes, Porphyromonadaceae_ and unclassified _Bacteria_ families were predictive of future _C. difficile_ colonization levels. Finally, challenging these mice with different strains of _C. difficile_ revealed that susceptible human-associated microbial communities were prone to severe disease independent of strain type. Taken together these results suggest that human-associated microbial communities can be recapitulated in germ-free mice and used to characterize dynamics of CDI. Because both healthy and diarrheal patients were susceptible to severe CDI, machine-learning models are useful to identify bacterial populations that allow colonization and contribute to the development of _C. difficile_ associated disease in humans. 
>>>>>>> master


\newpage

### Introduction
_Clostridium difficile_ is an opportunistic pathogen of the human lower gastrointestinal tract. _C. difficile_ forms spores that can persist on abiotic surfaces and are not readily killed by ethanol-based hand-sanitizers, putting hospital patients particularly at risk. Indeed, ~12% of hospital acquired infections in the United States are due to _C. difficile_ and result in up to 15,000 deaths annually [@Lessa2015]. Disruption of the native microbial community is the most common risk factor for development of _C. difficile_ infection (CDI) [@Britton2012]. Antibiotic use and inflammatory bowel diseases are associated with loss of colonization resistance to _C. difficile_ through the loss of potentially protective bacterial families such as _Barnesiella_ and _Lachnospiraceae_ in both mouse models and human association experiments [@Schubert2014;@Vincent2013]. The composition of the community is clearly important for the acquisition, resistance and treatment of _C. difficile_ infection, as giving patients a healthy fecal microbiome transplant is the most effective treatment for this disease [@Seekatz2014]. The precise mechanisms of colonization resistance and restoration of a healthy community are yet to be discovered.
  
Murine models to study CDI typically rely on treating conventionally-raised mice with antibiotics either in drinking water or by injection to induce susceptibility [@Buffie2011;@Theriot2011]. This model provides a convenient way to study _C. difficile_ pathogenesis and virulence factors. Numerous microbiome studies have been performed using this model to determine the antibiotic classes [@Schubert2015], starting microbial community [@Reeves2011] and metabolites [@Theriot2013] that impact development and severity of _C. difficile_ infection. While informative, these studies are somewhat removed from human disease because they only examine mouse-associated microbial communities. 
	
Gnotobiotic or germ-free mouse models have been used for a range of studies of CDI, including assessment of species-specific interactions between _C. difficile_ and competing microbial community members [@Reeves2012], analysis of nutrient restriction [@Ng2013], _in vivo_ transcriptomics of _C. difficile_ and examination of host immune response to CDI [@Pawlowski2010]. Further, CDI therapeutics such as antibiotics and fecal microbiota transplants have been tested extensively in a gnotobiotic-piglet or piglet-to-gnotobiotic-mouse model of disease [@Kim2014;@Diao2016]. Pigs have a longer digestive tract with components more similar to humans than mice and are typically infected by strains typical in human infection [@Steele2010]. However, the murine and porcine microbiomes typically do not resemble those of the human gut. 
  
<<<<<<< HEAD
The power of the gnotobiotic models to study CDI has been further realized by first inoculating germ-free mice and piglets with human stool microbes. In one study, germ-free piglets were acutely colonized with human feces for one week and then treated with tigecycline. After challenge with _C. difficile_ none of the antibiotic-treated piglets succumbed to infection, while some of the untreated human-colonized pigs did (11). In another study, germ-free mice colonized with human feces were bred over several generations to create a cohort of mice with identical human-derived microbiomes (14). These mice were subsequently treated with a five-antibiotic cocktail to induce dysbiosis and then were successfully colonized by _C. difficile_ (14).  While informative, these studies were limited in their use of only one human donor as input inoculum. In order to best understand the impact of _C. difficile_ pathogenesis on human disease, we must have a laboratory model that allows for study of a variety of human-derived microbiomes. 

Here we designed a laboratory model that allows for studying and modeling microbial coumminity interactions with _C. difficile_ infection in mice with human-derived microbiomes. To test the impact of individual human microbiomes on CDI, we colonized germ-free mice with 16 different human stool donors. We then characterized human-associated microbiome response to _C. difficile_ challenge. Additionally, the use of machine-learning models allowed us to build a predictive model that classified “at-risk” microbiomes prior to infection with _C. difficile_. These findings show that human-associated microbiomes can be at risk for CDI even in the absence of antibiotics and that study of mice colonized with human feces provides a range of clinical outcomes. 


### Results 
**Germ-free mice inoculated with human feces as model for _C. difficile_ infection.** To generate mice with human-derived microbiomes, we inoculated one cage of gnotobiotic C57/BL6 mice with one of 16 different human fecal donors. Five donors were patients that had diarrhea that was not attributable to _C. difficile_ infection while 11 donors were healthy at time of donation. Stool from a patient that was colonized with virulent _C. difficile_ was used as a positive control. After inoculation with human stool, mice were allowed to equilibrate for 14 days. Prior to infection, stool samples were taken from each mouse to establish baseline. Then, the _C. difficile_ strain isolated from the positive control patient’s sample (strain 430) was used to infect each mouse with 100 spores. Mice were monitored for weight loss and clinical signs of disease.  Fecal samples were taken to enumerate _C. difficile_ CFU and for microbiome analysis every day for up to 10 days post-infection (Fig 1A). To ensure that the donors we selected represented a diverse array of human microbiomes, we sequenced the 16S rRNA genes from donor fecal inocula. Ordination of the distances between donor communities showed that the donors each had distinctly different communities, independent of whether the sample came from a sick or healthy person (Fig 1B). Likewise, the starting microbial communities of the mice on day 0 were characterizing by sequencing of fecal pellets DNA prior to infection. Ordination of all of the mouse communities on day 0 shows that mice were similar to each other within each cage and donor, but distinct from other donors (Fig 1C). This result confirmed that human-associated microbes were able to colonize gnotobiotic mice and provide distinct initial communities to test _C. difficile_ dynamics. 

**need to put in a line to reference supplemental table with RA in it**

**_C. difficile_ infection in mice with human-derived microbiota cause a range of outcomes.** _C. difficile_ colonization was monitored by daily plating of stool pellets for _C. difficile_ CFU. Nearly all of the mice were colonized to 10^5^ – 10^7^ CFU by one day post-infection and remained colonized at that level until the end of the experiment (Fig 2A). As one indicator of disease, mouse weights were taken each day post-infection and weight-loss was monitored alongside clinical signs of disease. When mice were judged to be too ill to continue they were humanely euthanized. Overall, disease phenotypes fell into two classes. Mice that became severely ill and lost 20% or more of their starting body weight within one to two days post-infection were classified as “severe” whereas mice that were colonized with _C. difficile_ but did not show signs of disease or severe weight loss were considered to have “mild” disease (Fig 2A, 2B).  Interestingly, _C. difficile_ was able to cause severe disease in both mice that had been colonized with healthy stool and those colonized with diarrheal stool, suggesting susceptibility to CDI is dependent on the composition of the starting microbiome and not associated with donor clinical metadata. 

**Microbes present in the gut prior to infection are predictive of _C. difficile_ colonization levels** Previous work in our group has demonstrated that the microbes present in the mouse microbiome prior to _C. difficile_ challenge can predict future colonization levels (cite Alyx). To determine if the human-associated microbiome in the mouse could similarly predict _C. difficile_ colonization we employed a Random Forest machine learning algorithm similar to the one used in previous studies. Relative abundance of OTUs with greater than 1% abundance at day 0 were used as input to the model. We found that the model explained 86% of the variance in colonization levels of _C. difficile_ (Fig 3A). By refining the model to include only the top predictive OTUs and training with one individual mouse per cage, the model explained 77%% of the variance. Partial dependency plots of the top 6 OTUs predictive of _C. difficile_ colonization reveal that colonization is positively associated with _Enterococcus_, _Akkermansia_ (~2%) and an unidentifiable feature consisting of strongly corrlated OTUs of _Enterobacteriaceae_(13.5%), _Anaeroplasma_ (3.35%), _Prevotella_ (4.10%), _Alloprevotella_ (3.55%), and _Porphyromonadaceae_ (1.2%) (Fig 3B). Colonization was negatively associated with the presence of _Blautia_, _Ruminococcaceae_, _Lachnospiraceae_, _Clostridium XVII_, _Clostridium XIVb_, _Clostridium XIVa_, _Clostridiales_, _Oscilibacter_, and an unidentfiable feature representing highly correlated OTUs of _Clostridium XVIII_ (2.55%), _Holdemania_ (1.45%), _Robinsiella_ (2.20%), and _Clostridium IV_ (1.5%) (Fig 3B) though none of the human-associated mouse communities were completely resistant to colonization by _C. difficile_ (Fig 2A, 3B). These results confirm that as in mouse microbiome studies, the human-associated gut community prior to challenge is predictive of _C. difficile_ colonization levels. 

**The microbiome predicts _C. difficile_ infection severity.** As _C. difficile_ colonization levels can be predicted by the microbiome composition on day 0, we postulated that severity of disease could also be predicted from day 0. __Using weight loss as a proxy__(? and clinical signs of disease ?) for severity, we built a Random Forest classification model based on relative abundance to determine which community members were associated with either mild or severe disease. In order to rule out the potential cage effect was confounding the effect of the microbiome on disease severity, the model was built with a single mouse per human source. We optimized the model using the AUC-RF algorithm to generate a Reciever Operating Characteristic (ROC) curve with maximal Area Under the Curve (AUC) (cite Calle 2011), as used previously in our group to classify colon cancer outcomes (cite Niel 2015 gen med). The optimized model used 10 OTUs and identified several that predicted increased ~~weightloss and subsequent~~ disease severity (Fig 4A, 4B). The populations that were most commonly found in mice that succumbed to _C. difficile_ infection were members of the _Clostridiaceae_, _Akkermansia_, _Turicibacter_, _Odoribacter_, _Ruminococcaceae_ and _Parabacteriodes_. Whereas, _Clostridium XIVa_ and _Lachnospiraceae_ were in higher abundance in mice with a mild _C. difficle_ infection. The AUC was calculated using the Out-Of-Bag (OOB) probabilities for each sample (Fig 4A). Validation using leave-one-source-out, leave-two-sources-out, and 100-iterations of training with one-mouse per source leaving one source out resulted in AUCs not significantly different from the OOB (Fig 4A). This model reveals that the levels of 10 OTUs on day 0 can predict disease outcome in mice with human-associated microbiomes. (probably also going to want a supplemental table for predictive OTUs for this and Fig 3)

**Susceptibility to infection is community-dependent.** _C. difficile_ strains are known to vary in disease phenotypes based both on pathogenicity factors of the strain and host microbiome composition (cite someone). To test the susceptibility of human-derived microbiomes to _C. difficile_ infection, two donor-mouse microbiomes that were previously shown to be resistant to _C. difficile_ strain 431 induced weight loss (Fig 2B) (DA00396, DA00430) and one that was suscpetible to severe disease (DA00578) were independently challenged with one of three unique _C. difficile_ strains (table or reference for toxin prod, etc). As done previously, _C. difficile_ stool CFU was enumerated daily and mouse weight-loss tracked. Infection of mice with resistant communities with virulent strain 458 or the weaker strain 299 resulted in colonization to levels near that of 431 (Fig 5A) and these mice did not experience severe weight loss or show clinical signs of disease (Fig 5B). In contrast, infection of mice harboring microbiomes susceptible to severe disease (DA00578, Fig 1B) with strain 395 resulted in colonization and severe weight loss by two days post-infection (Fig 5A, 5B). Strain 395 is similarly virulent as strain 431, and shows a similar infection phenotype in these mice. These preliminary results suggest that human-associated microbiomes may be more predictive of disease severity than the _C. difficile_ strain. 
=======
The power of the gnotobiotic models to study CDI has been further realized by first inoculating germ-free mice and piglets with human stool microbes. In one study, germ-free piglets were acutely colonized with human feces for one week and then treated with tigecycline. After challenge with _C. difficile_ none of the antibiotic-treated piglets succumbed to infection, while some of the untreated human-colonized pigs did [@Kim2014]. In another study, germ-free mice colonized with human feces were bred over several generations to create a cohort of mice with identical human-derived microbiomes [@Collins2015]. These mice were subsequently treated with a five-antibiotic cocktail to induce dysbiosis and then were successfully colonized by _C. difficile_ [@Collins2015].  While informative, these studies were limited in their use of only one human donor as input inoculum.  

Here we designed a laboratory system that allows for studying and modeling microbial coumminity interactions with _C. difficile_ infection in mice with human derived microbiomes. To test the impact of individual human microbiomes on CDI, we colonized germ-free mice with 16 different human stool donors. We then characterized human-associated microbiome response to _C. difficile_ challenge. Further, we built a predictive model based on bacterial composition that classified “at-risk” microbiomes prior to infection with _C. difficile_. These findings show that human-associated microbiomes can be at risk for CDI even in the absence of antibiotic-induced dysbiosis. Finally, our results indicate that study of mice colonized with human feces can replicate a range of clinical outcomes. 


### Results 
**Germ-free mice inoculated with human feces as model for C. difficile infection.** To generate mice with human-derived microbiomes, we inoculated one cage each of gnotobiotic C57/BL6 mice with one of 16 different human fecal donors. Five donors were patients that had diarrhea that was not attributable to _C. difficile_ infection while 11 donors were healthy at time of donation. Stool from a patient that was colonized with virulent _C. difficile_ was used as a positive control. After inoculation with human stool, mice microbiomes were allowed to equilibrate for 14 days. Prior to infection, stool samples were taken from each mouse to establish baseline microbiome composition. Then, 100 spores of _C. difficile_ strain 430 (isolated from the positive control patient) were used to orally infect each mouse. Mice were monitored for weight loss and clinical signs of disease.  Fecal samples were taken to enumerate _C. difficile_ CFU and for microbiome analysis each day up to 10 days post-infection (Fig 1A). To ensure that the donors we selected represented a diverse array of human microbiomes, we sequenced the 16S rRNA genes from donor fecal inocula. Ordination of the distances between donor communities showed that the donors each had distinctly different communities, independent of whether the sample came from a sick or healthy person (Fig 1B). Likewise, the microbial communities of the mice on day 0 were characterized by sequencing of fecal pellet DNA prior to infection (Table S1). Ordination of day 0 communities shows that mice were similar to each other within each cage and donor, but distinct from other donors (Fig 1C). This result confirmed that human-associated microbes were able to colonize gnotobiotic mice and provide various initial communities to test _C. difficile_ dynamics. 

**C. difficile infection in mice with human-derived microbiota cause a range of outcomes.** _C. difficile_ colonization was monitored by daily plating of stool pellets for _C. difficile_ CFU. Nearly all of the mice were colonized to 10^5^ – 10^7^ CFU by one day post-infection and remained colonized at that level until the end of the experiment (Fig 2A). As one indicator of disease, mouse weights were taken each day post-infection and weight-loss was monitored alongside clinical signs. When mice were judged to be too ill to continue they were humanely euthanized. Overall, disease phenotypes fell into two classes. Mice that became severely ill and lost 20% or more of their starting body weight within one to two days post-infection were classified as “severe” whereas mice that were colonized with _C. difficile_ but did not show signs of disease or severe weight loss were considered to have “mild” disease (Fig 2A, 2B).  Interestingly, _C. difficile_ was able to cause severe disease in both mice that had been colonized with healthy stool and those colonized with diarrheal stool, suggesting susceptibility to CDI is dependent on the composition of the starting microbiome and not associated with donor clinical metadata. 

**Nick will be updating the next two sections**
**Microbes present in the gut prior to infection are predictive of _C. difficile_ colonization levels** Previous work in our group has demonstrated that the microbes present in the mouse microbiome prior to _C. difficile_ challenge can predict future colonization levels [@Schubert2015]. To determine if the human-associated microbiome in the mouse could similarly predict _C. difficile_ colonization we employed a Random Forest machine learning algorithm similar to the one used in previous studies. Relative abundance of OTUs with greater than 1% abundance at day 0 were used as input to the model. We found that the model explained X% of the variance in colonization levels of _C. difficile_ (Fig 3A). By refining the model to include only the top X predictive OTUs, the model explained X% of the variance. Partial dependency plots of the top X OTUs predictive of _C. difficile_ colonization reveal that colonization is positively associated with _Bacteriodetes_, _Parabacteriodes_ and _Porphyromonadaceae_ (really?) (Fig 3B). Colonization was negatively associated with the presence of _Lachnospiraceae_ (Fig 3B) though none of the human-associated mouse communities were completely resistant to colonization by _C. difficile_ (Fig 2A, 3B). These results confirm that as in mouse microbiome studies, the human-associated gut community prior to challenge is predictive of _C. difficile_ colonization levels. 
>>>>>>> master

**The microbiome predicts _C. difficile_ infection severity.** As _C. difficile_ colonization levels can be predicted by the microbiome composition on day 0, we postulated that severity of disease could also be predicted from day 0. Using weight loss as a proxy for severity, we built a Random Forest classification model based on relative abundance to determine which community members were associated with either mild or severe disease. We optimized the model using the AUC-RF algorithm to generate a Reciever Operating Curve (ROC) with maximal Area Under the Curve (AUC) [@Calle2011], as used previously in our group to classify colon cancer outcomes [@Baxter2016]. The optimized model used X number of OTUs and identified several that predicted increased weightloss and subsequent disease severity (Fig 4A, 4B). The populations that were most commonly found in mice that succumbed to _C. difficile_ infection were members of the _Clostridiales_, _Bacilliales_, _Streptococcus_ (Lactobacilles?) and _Bacteriodes_. (need more OTUs here, or number that were used for model and then group by rank). The AUC was calculated using the Out-Of-Bag (OOB) error for each sample (Fig 4A). Cross-validation using 10-fold, leave-one-mouse-out and leave-one-cage-out resulted in AUCs not significantly different from the OOB (Fig 4A). This model reveals that the levels of X OTUs on day 0 can predict disease outcome in mice with human-associated microbiomes. (probably also going to want a supplemental table for predictive OTUs for this and Fig 3)

### Discussion

Study of human-associated microbiomes and their interactions with _C. difficile_ has been limited by the lack of an appropriate model and available samples. Here we colonized 16 groups of mice with human donor stool from patients with different clinical profiles or healthy volunteers. This created a diverse set of human-associated mouse microbiomes that allowed for testing _C. difficile_ infection dynamics and community interactions. Our results showed that both healthy and diarrheal-associated human communities were susceptible to severe disease. Additionally, our results confirmed previous findings that resistance to _C. difficile_ disease is dependent on the entire microbial community and not just one or two taxa [@Schubert2015]. Further, we were able to learn classification models that identified specific OTUs within susceptible communities that predispose mice to infection. 

Here we colonized germ-free mice with a variety of human stool communities and allowed these communities to colonize the mice for 2 weeks, prior to _C. difficile_ challenge. It is important to note that the mouse gut environment is different from that of the human, and as such exerts a selective pressure on the human stool microbial community that stabilizes over the 2 weeks. Thus these mice cannot be considered "humanized" but rather colonized by human-associated microbes. Interestingly, clinical status was not associated with proclivity to severe _C. difficile_ infection as some healthy donor mice were susceptible. A surprising finding was that none of the human-associated mouse communities were completely resistant to colonization by _C. difficile_. This is in contrast to numerous reports of mouse microbiome studies [@Schubert2015] and the knowledge that in general healthy humans without gut perturbations are resistant to _C. difficile_ infection [@Heinlen2010]. On the other hand, it is known that humans, especially health care employees, can be asymptomatic carriers of _C. difficile_ [@FuruyaKanamori2015]. This may explain the fact that the majority of our groups of mice were colonized by _C. difficile_ but did not display outward signs of illness or infection-associated weight loss (Fig 2). 

•	Discuss prediction methods and outcomes - Nick - we have bugs that predict severity, but do we have bugs that predict mild/asymptomatic? probably, could discuss here
•	Discuss potential mechanisms for interesting OTUs - Nick

It has been recently reported that carriage of a non-toxigenic strain of _C. difficile_ can prevent additional colonization or recurrence by a pathogenic strain, suggesting a role of niche-filling or immunity this pre-colonization provides [@Gerding2015]. One interesting experiment would be to recapitulate this finding in human-associated mouse microbiomes by making a susceptible community resistant to infection. Further work with human-associated communities in mice and mathematical modeling will allow for uncovering of complex human-associated disease processes and dynamics as they relate to human _C. difficile_ infection.  


### Materials and Methods

**Mice** The experiments in this study used 6-8 week-old male and female gnotobiotic C57/BL6 mice. The mice were born and maintainted in a completely sterile environment in the Germ Free Mouse Facility at the University of Michigan. The University Committee on Use and Care of Animals approved all protocols and experiments conducted in this study.  

<<<<<<< HEAD
**Human samples** Donor stool samples were chosen from a sample bank collected from patients from October 2010 to November 2012 at the University of Michigan. Sample collection methods and study was approved by the Institutional Review Board at the University of Michigan. Stool samples from University of Michigan Health System patients were submitted to the microbiology lab for _C. difficile_ testing. _C. difficile_ presence was measured using **WHATEVER- Cdiff quik check? toxin AB?**. Samples from these patients were stored in Cary-Blair medium at -80C. All patients consented to participation in the study. Donor stool for this study was selected from one patient that tested positive for _C. difficile_ infection and five patients that presented with diarrhea and tested negative for _C. difficile_. Healthy donor stool was collected from adult volunteers living in the Ann Arbor, MI area. 100ul of stool slurry was gavaged into each mouse. 
=======
**Human samples** Donor stool samples were chosen from a sample bank collected from patients from October 2010 to November 2012 at the University of Michigan Health System. Sample collection methods and the study were approved by the Institutional Review Board at the University of Michigan. Stool samples from University of Michigan Health System patients were submitted to the microbiology lab for _C. difficile_ testing. _C. difficile_ presence was measured using C.Diff Quik Chek Complete Assay (Techlab). Fecal samples from these patients were stored in Cary-Blair medium at -80C. All patients consented to participation in the study. Donor stool was selected from one patient that tested positive for _C. difficile_ infection and five patients that presented with diarrhea and tested negative for _C. difficile_. Healthy donor stool was collected from adult volunteers living in the Ann Arbor, MI area. 100ul of stool slurry was gavaged into each mouse. 
>>>>>>> master

**_Clostridium difficile_ and sample processing** For all experiments, 100 spores of _Clostridium difficile_ were used to orally infect mice as described previously [@Schubert2015]. _C. difficile_ strain 431 is an 027 ribotype clinical isolate isolated from a patient with severe CDI [@Carlson2013]. After gavage, the remaining inoculum was diluted and plated to confirm infection dose. Bacteria were cultured anaerobically on TCCFA plates and incubated at 37C [@Bliss1997]. 
During infection, two fresh stool pellets were collected from each mouse daily. One pellet was resuspended in anaerobic phosphate-buffered saline and diluted and plated on TCCFA plates to enumerate _C. difficile_ CFU. The other pellet was frozen at -80C until the end of the experiment. 

**DNA isolation and sequencing** Bacterial DNA was isolated from banked donor stool and each mouse stool pellet using the MO-BIO PowerSoil DNA isolation kit. This DNA was used as template for amplification of the V4 region of the 16S rRNA gene and fragments were sequenced on an Illumina MiSeq as previously described [@Kozich2013]. 

•	Sequence curation/mothur, 
•	data analysis, 
•	code + availability of data(SRA) and code (Github) - Nick 

##Acknowledgments## 
We would like to thank members of the Schloss lab and Jhansi Leslie for experimental and analytical advice. We would also like to thank the Microbial Systems Laboratory at the University of Michigan for sequencing assistance. 


\newpage

###Figure Legends###

**Figure 1. Germ-free mice inoculated with human feces as a model for _C. difficile_ infection.** A) Experimental design. Stool from 16 healthy, diarrheal and CDI patients were independently inoculated into 3-4 germ-free mice by oral gavage. 14 days later mice were orally infected with 100 spores of _C. difficile_ strain 431. Weight and stool CFU were monitored for up to 10 days post infection. B) NMDS ordination of donor stool communities prior to inoculating mice. Each point represents one donor and are colored by clinical diagnosis. C) NDMS ordination of the stool communities on day 0. Each symbol represents one mouse and is colored by donor. Circles represent mice that experienced mild disease and triangles represent those that suffered severe disease. 

**Figure 2. _C. difficile_ infection results in mild or severe disease.** A) _C. difficile_ CFU was enumerated by plating of mouse stool pellets daily. Each point represents a mouse and the lines represent the median CFU in each group. Error bars are interquartile ranges. Red lines and points correspond to mice that succumbed to severe disease whereas black lines and points correspond to mice that had mild or no disease. B) Mouse weights were recorded and daily percent weight loss calculated for each mouse. Data is presented as the median of each group and interquartile ranges. Mice that succumbed to severe infection typically lost a significant amount of weight by day 1 or 2 post infection. Red lines correspond to severely ill mice, black to mice with mild disease. 

**Figure 3. Random Forest prediction of _C. difficile_ colonization level.** A) OTUs present in at least 10% of samples day 0 were used to predict median log~10~ CFU of _C. difficile_ after colonization. B) Partial dependency plots of the top six predictive OTUs. Line displays the partial dependence of log~10~ CFU on the relative abundance of each predictive OTU. Each median log~10~ CFU is plotted against its relative abundance for each predictive OTU. 

**Figure 4. Random Forest prediction of CDI severity.** OTUs present in at least 10% of samples day 0 were used to predict disease severity. OTUs were chosen such that they were not predictive of cage or donor. Predictive classification tested via 10-fold (gray), leave-one-cage-out (purple dashed) or leave-one-mouse-out (blue dashed) models are displayed in (A). B) Partial dependency plots of most predictive OTUs. Line displays the partial dependence of log~10~ CFU on OTU relative abundance. Points are the OTU relative abundance of each mouse colored by outcome (red, severe, black, mild). 

####Supplement####
**Table S1: Mouse day 0 communities by donor genera (avg + stdev of cage)**



\newpage

## References
